src/Datadog.Trace.AspNet/TracingHttpModule.cs

```diff
@@ -1,42 +1,40 @@
-#if !NETSTANDARD2_0
 using System;
 using System.Web;
-using Datadog.Trace.ClrProfiler.Interfaces;
-using Datadog.Trace.ClrProfiler.Models;
+using Datadog.Trace.ClrProfiler;
```

Comment:
Can you edit `src/Datadog.Trace/Util/UriHelpers.cs` so that it is defined inside the `Datadog.Trace` namespace instead of the `Datadog.Trace.ClrProfiler` namespace? Seeing this outside of the integrations module bothers me.

Reply:
Thanks! Fixed in 1cac15bff08f95fcb3cd19c0255b8e8c4b641235.